doc/api/process.md

```diff
@@ -928,6 +928,63 @@ console.log(process.env.test);
 
 `process.env` is read-only in [`Worker`][] threads.
 
+## process.allowedEnvironmentNodeFlags
```

Comment:
It seems this section should go between `## process.abort()` and `## process.arch`, ABC-wise.

Reply:
ok